pkgs/games/gshogi/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, buildPythonApplication, fetchFromGitHub
+, gtk3, gobjectIntrospection
+, wrapGAppsHook, python3Packages }:
+
+buildPythonApplication rec {
+  pname = "gshogi";
+  version = "0.5.1";
+  name = "${pname}-${version}";
```

Comment:
This line is not necessary for python apps since `name` is derived automatically. Please delete.